### PR TITLE
refactor and add sites

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

- Add shell routes for sites
- refactor regions into new file
- add sites routes in dummy client
- add sites routes in database client

These changes are forward compatible, so wont break current API + UI

Changed from
Screenshot 2024-07-26 at 11 27 43

to 

Screenshot 2024-07-26 at 11 27 50
It will also have auth on all routes

Helps with #61 
## How Has This Been Tested?

CI tests + ran locally

- [ ] Yes

## Checklist:

- [ ] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/india_api/internal/config/env.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def __init__(self) -> None:
 class Config(EnvParser):
     """Config for the application."""
 
-    SOURCE: str = "indiadb"
+    SOURCE: str = "dummydb"
```
Comment (braddf) [inline src/india_api/internal/config/env.py:59]: Was this change intentionally left in? Not sure if this is 100% being set in the env but probs safer to have the default as the real/prod value in this case..?

Comment (peterdudfield) [reply src/india_api/internal/config/env.py:59]: Yea my mistake, I'll change that bk

Comment (peterdudfield) [reply src/india_api/internal/config/env.py]: This should be tidied up now, but I'm on my phone so I can't quite see

File: src/india_api/internal/service/__init__.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
-from .server import server, get_db_client
+from .server import server
+from .database_client import get_db_client
```
Comment (braddf) [inline src/india_api/internal/service/__init__.py:2]: Unused import now this is split and not in the array..?

File: src/india_api/internal/service/server.py [hunk 3]
```diff
@@ -1,48 +1,80 @@
...
-# TODO: add scopes for granular access across APIs
-# auth = Auth(domain=os.getenv('AUTH0_DOMAIN'), api_audience=os.getenv('AUTH0_API_AUDIENCE'), scopes={'read:india': ''})
+tags_metadata = [
+    {
+        "name": "API Information",
+        "description": "General API information,",
+    },
+    {
+        "name": "Sites",
+        "description": "A site is a specific location, for example (52.15,1.25) in latitude and longitude. "
```
Comment (braddf) [inline src/india_api/internal/service/server.py:28]: Maybe "specific point location"? And missing space in lat/long numbers

File: src/india_api/internal/service/server.py [hunk 4]
```diff
@@ -1,48 +1,80 @@
...
+        "Each site will have one source of energy "
+        "and there is forecast and generation data for each site. ",
+    },
+    {
+        "name": "Regions",
+        "description": "A region is an area of land e.g. a Alaska in the USA. "
```
Comment (braddf) [inline src/india_api/internal/service/server.py:34]: Typo "e.g. ~a~ Alaska in the USA"

**Review by braddf — CHANGES_REQUESTED** [review]
Couple of tiny comments but good to merge then!

File: src/india_api/internal/service/__init__.py (hunk 2 above)
Comment (peterdudfield) [reply src/india_api/internal/service/__init__.py:2]: to be honest, im not quite sure what this file does, do perhaps ill put it back to what it was

**Review by braddf — APPROVED** [review]

**Pull request merged**